integration-tests/fhir/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+# Quarkus
+#
+quarkus.ssl.native=true
+#
+# Quarkus :: Camel
+#
+quarkus.camel.disable-jaxb=false
+#
+# Quarkus :: Camel :: FHIR
+#
+camel.fhir.disable-dstu2=true
+camel.fhir.disable-dstu3=false
+camel.fhir.disable-r4=true
```

Comment:
Why are those two disabled? A comment would be nice.

Reply:
BTW, there is https://github.com/apache/camel-quarkus/issues/285 about the `disable*` names.

Reply:
Done